spec/rails_helper.rb

```diff
@@ -73,6 +74,13 @@
   config.include FeatureFlagHelper
   config.include ViewComponent::TestHelpers, type: :component
   config.include Capybara::RSpecMatchers, type: :component
+  config.include ViewComponent::SystemTestHelpers, type: :component_sys_test
+
+  config.before(:each, type: :component_sys_test) do
```

Comment:
Might be worth adding a comment linking the issue were we found this fix in here, just incase people come across it and wonder why.